### PR TITLE
[doc] Cdc for cassandra event behavior explanation

### PR DESCRIPTION
**Pull request by mendonk**

### CODE REVIEW

**Comment by vroyer** [comment]
@mendonk I competed your doc, please check my english, thanks.

**Review by mendonk — COMMENTED** [review]
@vroyer a few small changes for organization, but it looks good to me.

**Comment by eolivelli** [comment]
@mendonk what's the status of this PR ?

File: docs/modules/ROOT/pages/cdc-cassandra-events.adoc [hunk 1]
```diff
@@ -0,0 +1,188 @@
+= CDC for Cassandra Events 
+
+The Datastax CDC for Cassandra agent pushes the mutation primary key for the CDC-enabled table into the Apache Pulsar events topic (also called the dirty topic). The messages in the data topic (or clean topic) are keyed messages where both the key and the payload are https://avro.apache.org/docs/current/spec.html#schema_record[AVRO records]:
```
Comment (msmygit) [inline docs/modules/ROOT/pages/cdc-cassandra-events.adoc:3]: Could we please globally change all the `Datastax` to `DataStax` (notice the upper case `S`) to preserve our branding! Thanks!

**Pull request merged**